data/creaturescripts/scripts/login.lua

```diff
@@ -33,6 +33,12 @@ function onLogin(player)
 		player:setStorageValue(PlayerStorageKeys.achievementsTotal, player:getAchievementPoints())
 	end
 
+	-- Bestiary tracker
+	local trackedBestiary = player:getTrackedBestiary(true)
```

Comment:
```suggestion
	local trackedBestiary = player:getTrackedBestiary()
```

I think the parameter was removed at some point